plugins/modules/zos_job_output.py

```diff
@@ -112,11 +112,31 @@
          The name of the batch job.
       type: str
       sample: HELLO
+    system:
+      description:
+         The job entry system that MVS uses to do work.
+      type: str
+      sample: STL1
     subsystem:
       description:
          The job entry subsystem that MVS uses to do work.
       type: str
       sample: STL1
+    cpu_time:
+      description:
+        Sum of the CPU time used by each job step, in microseconds.
+      type: int
+      sample: 5
+    execution_node:
+      description:
+        Execution node.
+      type: str
+      sample: "STL1"
+    origin_node:
+      description:
+        Origin node (node of submittal).
```

Comment:
I took part of this description from ZOAU docs but adapted it to be more user friendly.
```suggestion
        Origin node that submitted the job.
```